sql/parser/expr.go

```diff
@@ -264,6 +275,20 @@ func (n QualifiedName) String() string {
 	return buf.String()
 }
 
+// QualifiedNames is a comma separated list of QualifiedNames.
```

Comment:
This comment should probably talk about `String()` since it's an array and not a `string`